src/libmongoc/doc/mongoc_client_command.rst

```diff
@@ -3,6 +3,12 @@
 mongoc_client_command()
 =======================
 
+.. warning::
+   .. deprecated:: 1.29.0
+
+      This function is deprecated and should not be used in new code.
+      This function is superseded by :symbol:`mongoc_client_command_simple()`.
```

Comment:
```suggestion
      Use :symbol:`mongoc_client_command_simple()` instead.
```
Imperative language + consistency with release notes. Also applies to the other doc pages.